tasks/main.yml

```
@@ -133,6 +133,17 @@
         _config_yml is changed or
         _sentry_conf_py is changed
 
+- name: Create Sentry users
+  shell: >
+    '
+    {{ sentry_install_dir }}/bin/sentry --config {{ sentry_install_dir }}/etc createuser --no-input
+     --email {{ item.email }}
+     --password {{ item.password }}
+    {{ item.superuser | ternary("--superuser", "--no-superuser", "") }}
+    '
+  with_items: "{{ sentry_users }}"
+  ignore_errors: true
```

Maybe instead of blanket ignoring all the errors, you could later check whether the error you were expecting is what was thrown?

```suggestion
  register: sentry_create_user_result
  ignore_errors: true

- name: Check if user creation command OK
  fail:
    msg: Could not create some Sentry users
  when: item.rc != 0 and "already exists." not in item.stderr 
  with_items: "{{ sentry_create_user_result.results }}"
```